utils/record/record.go

```diff
@@ -167,23 +204,21 @@ func NewAPIFactorySettings() api.Settings {
 
 // Send notifications for triggered event if user is subscribed
 func (e *EventRecorderAdapter) sendNotifications(object runtime.Object, opts EventOptions) error {
+	notificationsAPI, err := e.apiFactory.GetAPI()
+	if err != nil {
+		return err
+	}
+	cfg := notificationsAPI.GetConfig()
 	logCtx := logutil.WithObject(object)
-	subsFromAnnotations := subscriptions.Annotations(object.(metav1.Object).GetAnnotations())
-	destByTrigger := subsFromAnnotations.GetDestinations(nil, map[string][]string{})
-
+	destByTrigger := cfg.GetGlobalDestinations(object.(metav1.Object).GetLabels())
+	destByTrigger.Merge(subscriptions.NewAnnotations(object.(metav1.Object).GetAnnotations()).GetDestinations(cfg.DefaultTriggers, cfg.ServiceDefaultTriggers))
 	trigger := translateReasonToTrigger(opts.EventReason)
-
 	destinations := destByTrigger[trigger]
 	if len(destinations) == 0 {
 		logCtx.Debugf("No configured destinations for trigger: %s", trigger)
 		return nil
 	}
 
-	notificationsAPI, err := e.apiFactory.GetAPI()
-	if err != nil {
-		return err
-	}
-
 	// Creates config for notifications for built-in triggers
 	triggerActions, ok := notificationsAPI.GetConfig().Triggers[trigger]
```

Comment:
notificationsAPI.GetConfig() can be replaced by the variable cfg as defined in line 211